docs/guides/nrfconnect_android_commissioning.md

```diff
@@ -0,0 +1,210 @@
+  > :information_source: One nRF52840 DK can be replaced with another [OpenThread Radio Co-Processor](https://openthread.io/platforms/co-processor) compatible device like nRF52840 Dongle
+- 1x smartphone with Android 8+
+- 1x PC with Ubuntu 20.04 and Docker installed, with a spare Wi-Fi card.<br>
+  > :information_source: While the article uses Ubuntu 20.04 as a reference, all the presented steps should be achievable using other popular operating systems.
+
+The essence of the commissioning process is composed of the following pieces:
+1. CHIPTool discovers the accessory device over BLE.
+2. CHIPTool establishes a secure channel to the device over BLE, sends CHIP operational credentials and Thread provisioning data.
+3. The accessory device joins a Thread network.
+
+BLE is only used during the commissioning phase and afterwards IP connectivity between the smartphone and the accessory device is needed to send operational messages. Since a typical smartphone doesn't have Thread radio built-in, extra effort is needed to prepare full-fledged test environment. The article describes how to turn a PC with spare Wi-Fi card and nRF52840 DK into a Thread border router forwarding Wi-Fi packets to the Thread network the accessory device operates in. The below diagram shows connectivity between network components necessary to allow communication between devices running CHIPTool and Lock applications:
+
+```
+               +--------------------+
+               |    Smartphone      |
+     +---------|  Android CHIPTool  |---------+
+     |         +--------------------+         |
+     |                                        | Wi-Fi
+     |                                        |
+     |                                +---------------+  Ethernet  +----------+
+     |                                |      PC       |------------| Internet |
+     |                                +---------------+            +----------+
+     |                                        |
+ BLE |                                        | USB
+     |                                        |
+     |                                +---------------+
+     |                                |  nRF52840 DK  |
+     |                                | OpenThread RCP|
+     |                                +---------------+
+     |                                        |
+     |         +--------------------+         | Thread
+     |         |    nRF52840 DK     |         |
+     +---------|  Lock Application  |---------+
+               +--------------------+
+```
+## Building and flashing OpenThread RCP firmware
+
+<details>
+<summary>Click here to expand</summary>
+<br>
+
+> :information_source: In case you want to use nRF52840 Dongle as an OpenThread Radio Co-Processor read [nRF52840 dongle support](https://github.com/openthread/openthread/blob/master/examples/platforms/nrf528xx/nrf52840/README.md#nrf52840-dongle-support-pca10059) and adjust the steps below accordingly.
```

Comment:
I think this link is not that helpful. It links to a section, which contains another link to another "more details" section, and there I end up on a page that asks me to install nRF Connect Programmer. I feel totally lost, so I guess we need to specify which steps should be adjusted how, or have a subsection that only mentions steps that are different.

Reply:
I'm not sure what to do about this. I think the article is long enough and I would prefer it not cover multiple boards etc. On the other hand, @LuDuda asked me to include info about the possibility to use the dongle. Can't we somehow mark some notes "For experienced users" or it's undesired. I think an experienced embedded developer shouldn't have problems understanding how to flash the dongle, so the link might be helpful for him...

Reply:
Let's not assume who is experienced and who is not. 
I don't have a problem with the information, I have a problem with the link being not that precise. You say "adjust the steps below accordingly" -- which steps? according to what information?

Reply:
I don't think we want to make this tutorial an information source on all related topics. If you think little information is more confusing than no information I'll remove the note.